src/pages/Dashboard/Dashboard.jsx

```diff
@@ -104,7 +112,15 @@ const Dashboard = () => {
             </Button>
           )}
           <div className={`${styles['side-bar']} ${markerInfoIsOpen ? styles['show-info'] : ''}`}>
-            <Slide className={styles['marker-info-slide']} direction="right" in={markerInfoIsOpen}>
+            <Slide
+              className={
+                adminIsLoggedIn
+                  ? styles['marker-info-slide-admin']
+                  : styles['marker-info-slide-general']
+              }
+              direction="right"
+              in={markerInfoIsOpen}
+            >
```

Comment:
Not sure if this is the right code block to comment on, but could we make the right sidebar slide out when the user clicks on the X button to close the sidebar? The left sidebar slides to open and close but the right sidebar only slides on open.